{"uid":"f39df9ce6dabe09c","name":"test_deploy_stable","fullName":"tests.integration.ha_tests.test_upgrade_from_stable#test_deploy_stable","historyId":"14659fe2e8ff20c0c1c1d36c1d97e7ab","time":{"start":1718156427716,"stop":1718156428407,"duration":691},"description":"Simple test to ensure that the PostgreSQL and application charms get deployed.","descriptionHtml":"<p>Simple test to ensure that the PostgreSQL and application charms get deployed.</p>\n","status":"broken","statusMessage":"juju.errors.JujuError: resolving ch:postgresql-k8s : selecting releases: charm or bundle not found for channel \"14/stable\", base \"arm64\"\navailable releases are:\n  channel \"14/candidate\": available series are: jammy\n  channel \"14/beta\": available series are: jammy\n  channel \"14/edge\": available series are: jammy","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: Optional[TResult] = func()\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 241, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 87, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 63, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 90, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 65, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 850, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 833, in _runtest_for\n    yield\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 878, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 183, in pytest_runtest_call\n    raise e\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 173, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1632, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 162, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/tests/integration/ha_tests/test_upgrade_from_stable.py\", line 37, in test_deploy_stable\n    await asyncio.gather(\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 1743, in deploy\n    res = await self.deploy_types[schema].resolve(entity, architecture,\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 529, in resolve\n    charm_url, origin = await self.charm_resolver(url, origin, force, series, model_conf)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 1918, in _resolve_charm\n    raise JujuError(f'resolving {url} : {result.error.message}')\njuju.errors.JujuError: resolving ch:postgresql-k8s : selecting releases: charm or bundle not found for channel \"14/stable\", base \"arm64\"\navailable releases are:\n  channel \"14/candidate\": available series are: jammy\n  channel \"14/beta\": available series are: jammy\n  channel \"14/edge\": available series are: jammy","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"event_loop","time":{"start":1718156427337,"stop":1718156427337,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1718156427337,"stop":1718156427715,"duration":378},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1718156427337,"stop":1718156427337,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1718156427716,"stop":1718156427716,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"Simple test to ensure that the PostgreSQL and application charms get deployed.","status":"broken","statusMessage":"juju.errors.JujuError: resolving ch:postgresql-k8s : selecting releases: charm or bundle not found for channel \"14/stable\", base \"arm64\"\navailable releases are:\n  channel \"14/candidate\": available series are: jammy\n  channel \"14/beta\": available series are: jammy\n  channel \"14/edge\": available series are: jammy","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: Optional[TResult] = func()\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 241, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 87, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 63, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 90, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 65, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 850, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 833, in _runtest_for\n    yield\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 878, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 183, in pytest_runtest_call\n    raise e\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 173, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1632, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 162, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/tests/integration/ha_tests/test_upgrade_from_stable.py\", line 37, in test_deploy_stable\n    await asyncio.gather(\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 1743, in deploy\n    res = await self.deploy_types[schema].resolve(entity, architecture,\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 529, in resolve\n    charm_url, origin = await self.charm_resolver(url, origin, force, series, model_conf)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 1918, in _resolve_charm\n    raise JujuError(f'resolving {url} : {result.error.message}')\njuju.errors.JujuError: resolving ch:postgresql-k8s : selecting releases: charm or bundle not found for channel \"14/stable\", base \"arm64\"\navailable releases are:\n  channel \"14/candidate\": available series are: jammy\n  channel \"14/beta\": available series are: jammy\n  channel \"14/edge\": available series are: jammy","steps":[],"attachments":[{"uid":"abcdc3d6ee7f1f9d","name":"log","source":"abcdc3d6ee7f1f9d.txt","type":"text/plain","size":128}],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"abort_on_fail::0","time":{"start":1718156428539,"stop":1718156428539,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1718156428547,"stop":1718156429204,"duration":657},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1718156429205,"stop":1718156429205,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1718156429205,"stop":1718156429205,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1718156429205,"stop":1718156429205,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"abort_on_fail"},{"name":"tag","value":"group(1)"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration.ha_tests"},{"name":"suite","value":"test_upgrade_from_stable"},{"name":"host","value":"runner"},{"name":"thread","value":"13709-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.ha_tests.test_upgrade_from_stable"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"9839932a55ab5245","status":"unknown","time":{}}],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":4,"skipped":0,"passed":0,"unknown":1,"total":5},"items":[{"uid":"f209e53417d8c03e","reportUrl":"../75//#testresult/f209e53417d8c03e","status":"broken","statusDetails":"juju.errors.JujuError: resolving ch:postgresql-k8s : selecting releases: charm or bundle not found for channel \"14/stable\", base \"arm64\"\navailable releases are:\n  channel \"14/candidate\": available series are: jammy\n  channel \"14/beta\": available series are: jammy\n  channel \"14/edge\": available series are: jammy","time":{"start":1718071295713,"stop":1718071296360,"duration":647}},{"uid":"87aba91b9a59f358","reportUrl":"../74//#testresult/87aba91b9a59f358","status":"broken","statusDetails":"juju.errors.JujuError: resolving ch:postgresql-k8s : selecting releases: charm or bundle not found for channel \"14/stable\", base \"arm64\"\navailable releases are:\n  channel \"14/candidate\": available series are: jammy\n  channel \"14/beta\": available series are: jammy\n  channel \"14/edge\": available series are: jammy","time":{"start":1718005192855,"stop":1718005193521,"duration":666}},{"uid":"d47ceea3e75fee16","reportUrl":"../73//#testresult/d47ceea3e75fee16","status":"unknown","time":{}},{"uid":"2cc6a711db0ec23d","reportUrl":"../72//#testresult/2cc6a711db0ec23d","status":"broken","statusDetails":"juju.errors.JujuError: resolving ch:postgresql-k8s : selecting releases: charm or bundle not found for channel \"14/stable\", base \"arm64\"\navailable releases are:\n  channel \"14/candidate\": available series are: jammy\n  channel \"14/beta\": available series are: jammy\n  channel \"14/edge\": available series are: jammy","time":{"start":1717813368274,"stop":1717813368611,"duration":337}}]},"tags":["asyncio","abort_on_fail","group(1)"]},"source":"f39df9ce6dabe09c.json","parameterValues":[]}